phpstan.neon.dist

```diff
@@ -0,0 +1,18 @@
+includes:
+    - 'vendor/phpstan/phpstan-phpunit/extension.neon'
+    - 'vendor/phpstan/phpstan-phpunit/rules.neon'
+
+parameters:
+    checkGenericClassInNonGenericObjectType: false
+    checkMissingIterableValueType: false
+    ignoreErrors:
+        - '#Parameter \#1 \$function of function call_user_func expects callable#'
```

Comment:
Call the `is_callable` function [here](https://github.com/M6Web/StatsdTagsPrometheusBundle/blob/b0d184c02f1dcf3e410cf328ae068e1699db2a24/Metric/Metric.php#L109) will fix this error